<launch>

  <!-- Include the shared resources launch file -->
  <include file="$(find omniroute_operation)/launch/shared_resources.launch" />

  <!-- Node start delays (sec) -->
  <arg name="single_T_maze_node_start_delay" default="0.0" />

  <!-- Single T-maze interface -->
  <node name="single_T_maze" pkg="omniroute_operation" type="single_T_maze"
    output="screen" launch-prefix="bash -c 'sleep $(arg single_T_maze_node_start_delay); $0 $@' " />

</launch>